src/Microsoft.AspNetCore.Mvc.ViewFeatures/MvcViewOptions.cs

```diff
@@ -87,6 +91,17 @@ public bool SuppressTempDataAttributePrefix
             set => _suppressTempDataAttributePrefix.Value = value;
         }
 
+        /// <summary>
+        /// Gets or sets a value that indicates whether the maxlength attribute should be rendered for compatible HTML input elements,
+        /// when they're bound to models marked with either
+        /// <see cref="StringLengthAttribute"/> or <see cref="MaxLengthAttribute"/> attributes.
+        /// </summary>
+        public bool MaxLengthAttributeRenderingEnabled
```

Comment:
Should be named `AllowRenderingMaxLengthAttribute`. See guidelines - https://github.com/aspnet/Mvc/blob/release/2.2/src/Microsoft.AspNetCore.Mvc.Core/Infrastructure/CompatibilitySwitch.cs#L37-L39

Reply:
Renamed